src/placement/pl_map.h

```diff
@@ -81,6 +81,10 @@ struct failed_shard {
 	uint8_t         fs_status;
 };
 
+#define	DF_FAILEDSHARD "shard_idx: %d, fseq: %d, tgt_id: %d, status: %d"
+#define	DP_FAILEDSHARD(x) (x).fs_shard_idx, (x).fs_fseq, \
```

Comment:
(style)  Macros with complex values should be enclosed in parentheses

Reply:
"DF_SHARD/DP_SHARD" seems enough